### PR TITLE
Username change

### PR DESCRIPTION
**Pull request by imax9000**

### CODE REVIEW

**Review by per1234 — COMMENTED** [review]
Hello @imax9000. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

After that, you can monitor the indexing of the library via its dedicated logs web page.
Information about that here:
https://github.com/arduino/library-registry/blob/main/FAQ.md#can-i-check-on-library-releases-being-added-to-library-manager

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/04/29 16:13:06 Loaded 6875 libraries from DB
Changing URL of library PID_v2 from https://github.com/gelraen/Arduino-PID-Library.git to https://github.com/imax9000/Arduino-PID-Library.git
Success!
```

**Pull request merged**
